### PR TITLE
Add Remix examples

### PR DESCRIPTION
**Pull request by bnusunny**
*Issue #, if available:*

https://github.com/awslabs/aws-lambda-web-adapter/discussions/399 

*Description of changes:*

Add two examples for Remix

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/remix-zip/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+### Build and Deploy
+
+Run the following commands to build and deploy the application to lambda. SAM CLI will use GNU Make to build the Remix application.
+
+```bash
+sam build
+sam deploy --guided
+```
+When the deployment completes, take note of RemixFunctionApi's Value. It is the API Gateway endpoint URL. 
+
```
Comment (valerena) [inline examples/remix-zip/README.md:32]: Most of the other examples (and the Remix-image example below) have a section in the README for local testing. You should add something here related to that. Like [SpringBoot](https://github.com/awslabs/aws-lambda-web-adapter/tree/main/examples/springboot-zip#local-test-with-sam-cli), using SAM CLI, or if Remix has its own way for local testing.

Comment (bnusunny) [reply examples/remix-zip/README.md:32]: Sure, added the section.

Comment (valerena) [reply examples/remix-zip/README.md:32]: You added a SAM local testing section to the README inside the image app (which also had a way to locally test with docker directly), but didn't add it in this file (-zip version)

Comment (bnusunny) [reply examples/remix-zip/README.md:32]: Updated

**Review by valerena — APPROVED** [review]

**Pull request merged**